serializer.rst

```diff
@@ -1239,6 +1239,67 @@ setting the ``name_converter`` setting to
         ];
         $serializer = new Serializer($normalizers, $encoders);
 
+snake_case to CamelCase
+~~~~~~~~~~~~~~~~~~~~~~~
+
+In Symfony applications is common to use camelCase to name properties. However
```

Comment:
```suggestion
In Symfony applications it is common to use camelCase to name properties. However
```

Reply:
Done while merging ... and added the `versionadded` directive too. Thanks!